docs/notebook-demos/reading-in-sentinel-data.ipynb

```diff
@@ -0,0 +1,225 @@
+{
```

Comment:
The biggest reason is that by getting it all into GeoPySpark you'll be able to work large amounts of data in a distributed fashion. In addition to being able to use GeoTrellis. I'll make a small section going over the benefits at the beginning of the document.